gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/explode/writer/PolicyWriter.java

```diff
@@ -45,9 +45,12 @@ public void write(Bundle bundle, File rootFolder) {
         Map<String, Service> services = bundle.getEntities(Service.class);
         services.values().parallelStream().forEach(serviceEntity -> writePolicy(bundle, policyFolder, serviceEntity.getParentFolder().getId(), serviceEntity.getName(), serviceEntity.getPolicyXML()));
 
-        Map<String, Policy> policies = bundle.getEntities(Policy.class);
-        policies.values().parallelStream().forEach(policyEntity -> writePolicy(bundle, policyFolder, policyEntity.getParentFolder().getId(), policyEntity.getName(), policyEntity.getPolicyDocument()));
-
+        Stream.of(
+                bundle.getEntities(Policy.class).values().stream(),
+                bundle.getEntities(GlobalPolicy.class).values().stream().peek(Policy.class::cast).collect(toList()).stream(),
+                bundle.getEntities(AuditPolicy.class).values().stream().peek(Policy.class::cast).collect(toList()).stream()
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "Stream.peek". ([squid:S3864](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS3864))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=128&issues=AWcPsqomd6oxh3R490_D&open=AWcPsqomd6oxh3R490_D)

[ISSUE_KEY]: AWcPsqomd6oxh3R490_D